python-susecloudimagenameparser.spec

```diff
@@ -0,0 +1,47 @@
+#
+# spec file for package susecloudimagenameparser 
+#
+# Copyright (c) 2022 SUSE LLC
+#
+
```

Comment:
Pre-amble missing

"""
All modifications and additions to the file contributed by third parties
remain the property of their copyright owners, unless otherwise agreed
upon. The license for this file, and modifications and additions to the
file, is the same license as for the pristine package itself (unless the
license for the pristine package is not an Open Source License, in which
case the license is the MIT License). An "Open Source License" is a
license that conforms to the Open Source Definition (Version 1.9)
published by the Open Source Initiative.
"""